tests/test_batch_edit.py

```diff
@@ -20,3 +22,213 @@ def test_change_tags():
 
         a.change_tags(query, "test", add=False)
         assert len(list(a.find_notes(query))) == 0
+
+
+def test_add_from_file(collection):
+    """Test adding a note from a Markdown file."""
+    with open("test.md", "w") as f:
+        f.write(
+            """model: Basic
+tags: marked
```

Comment:
Is there a way to have this _not_ look really ugly? I don't know, and it's not very important. I did find other people discuss this, e.g. https://stackoverflow.com/questions/2504411/proper-indentation-for-multiline-strings. There the `textwrap.dedent` was proposed as a solution. 🤷🏻